lib/theme_check/language_server/completion_providers/tag_completion_provider.rb

```diff
@@ -6,7 +6,9 @@ class TagCompletionProvider < CompletionProvider
       def completions(content, cursor)
         return [] unless can_complete?(content, cursor)
         partial = first_word(content) || ''
-        ShopifyLiquid::Tag.labels
+        labels = ShopifyLiquid::Tag.labels
+        labels += ShopifyLiquid::Tag.end_labels if partial.start_with?('end')
```

Comment:
I'm curious why you need the `start_with?` here. Why not return all possible completions, and let the rest of the completion engine deal which one is relevant?

Reply:
Yeah. I was on the fence with this one. Wanted to hide the 10ish end\* options if you were scrolling down.

Reply:
I'll remove that. It's superfluous and debatable.